pkg/dht/dht.go

```diff
@@ -23,6 +23,7 @@ type DHT interface {
 	Bootstrap(ctx context.Context) error
 	Ping(ctx context.Context, node proto.Node) (proto.Node, error)
 	FindNode(ctx context.Context, ID NodeID) (proto.Node, error)
+	FindNearNodes(ctx context.Context, ID NodeID) ([]*proto.Node, error)
```

Comment:
isn't this what GetNodes is for? perhaps `start` should be a `NodeID`

Reply:
Removed.